docs/api/connection.rst

```diff
@@ -346,3 +346,207 @@ Connection
 
 .. _BigInt:
     https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt
+
+
+.. _edgedb-js-api-pool:
+
+Pool
+==========
+
+.. js:class:: Pool(options: PoolOptions)
+
+    A connection pool is used to manage a set of connections to a database.
+    Since opening connections is an expensive operation, connection pools are used
+    to maintain and reuse connections, enhancing the performance of database
+    interactions.
+
+    Pools can be created using the static constructor :js:meth:`Pool.create` of
+    js:class:`~edgedb.Pool`.
+
+    .. code-block:: js
+
+        // Use the Node.js assert library to test results.
+        const assert = require("assert");
+        const edgedb = require("edgedb");
+
+        async function main() {
+        const pool = await edgedb.Pool.create({
+            connectOptions: {
+                user: "edgedb",
+                host: "127.0.0.1",
+            },
+        });
+
+        await pool.initialize();
+
+        try {
+            let data = await pool.fetchOne("SELECT [1, 2, 3]");
+
+            // The result is an Array.
+            assert(data instanceof Array);
+            assert(typeof data[0] === "number");
+            assert(data.length === 3);
+            assert(data[2] === 3);
+        } finally {
+            // in this example, the pool is closed after a single
+            // operation; in real scenarios a pool is initialized
+            // at application startup, and closed at application shutdown
+            await pool.close();
+        }
+        }
```

Comment:
This bracket is weird. I guess it should be removed?

Reply:
No, indentation was wrong, the bracket is right. I am fixing it. :)